### PR TITLE
[CHORE] Code review pt2

### PR DESCRIPTION
**Pull request by MikeVedsted**
Todos that I could easily fix have been included.
Issues have been created for todos that require investigation or are more than quick fixes

### CODE REVIEW

File: client/src/pages/LandingPage/index.tsx [hunk 1]
```diff
@@ -9,70 +8,64 @@ import { eventCategories } from '../../util/constants/eventCategories'
...
       const response = await axios({
         method: 'GET',
-        url: 'http://localhost:5000/api/v1/users'
+        url: '/api/v1/users'
       })
       const data = response.data.length
       setUsers(data)
     } catch (error) {
-      setError(error.message)
+      setUsers('Many')
```
Comment (Rostislav013) [inline client/src/pages/LandingPage/index.tsx:27]: Is it good idea to get all users here? 
And do we need it?

Comment (MikeVedsted) [reply client/src/pages/LandingPage/index.tsx:27]: Well, if we access to that route in the backend, it won't work... We could make a route to just return the user count maybe?

Comment (Rostislav013) [reply client/src/pages/LandingPage/index.tsx:27]: Just any random word(number) heh

**Review by MarinaFroes — APPROVED** [review]
I think it's ok.

**Pull request merged**